reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -459,6 +462,19 @@ public HttpHeaders requestHeaders() {
 		throw new IllegalStateException("request not parsed");
 	}
 
+	@Override
+	public Map<String, List<String>> queryParams() {
+		if (queryParamsMap == null) {
+			queryParamsMap = Collections.unmodifiableMap(parseQueryParams(this.nettyRequest));
+		}
+		return queryParamsMap;
+	}
+
+	private Map<String, List<String>> parseQueryParams(HttpRequest request) {
+		QueryStringDecoder decoder = new QueryStringDecoder(request.uri());
+		return decoder.parameters();
+	}
```

Comment:
in order to avoid mixing public and private methods, what do you think about moving the private `parseQueryParams`  method just before the static inner classes that are declared in the end of the class ?